src/lib/queries/queryKeys.ts

```diff
@@ -3,5 +3,8 @@ export const USER_QUERY_KEY = (userId: string): string[] => ['user', userId];
 export const USERS_QUERY_KEY = 'users';
 
 // Diving cylinder set related query keys
-
+export const CYLINDER_SET_QUERY_KEY = (setId: string): string[] => [
+  'set',
```

Comment:
Ehkä tää vois olla kuitenkin
```suggestion
  'divingCylinderSet',
```